libcxx/utils/libcxx/test/format.py

```diff
@@ -172,6 +172,7 @@ def parseScript(test, preamble):
                 f"{compileFlags} "
                 "-Wno-reserved-module-identifier -Wno-reserved-user-defined-literal "
                 "-fmodule-file=std=%T/std.pcm " # The std.compat module imports std.
+                "-fexperimental-modules-reduced-bmi "
```

Comment:
Is the intent to have people using a `-fexperimental-foo` flag whenever they compile with C++20 modules enabled?

Reply:
Yes